<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Room_MainWid</class>
 <widget class="QWidget" name="Room_MainWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>583</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="titleWid" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>机房列表</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>26</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>19</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="roomLabel">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>所有设备</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>31</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="6">
       <widget class="QToolButton" name="upBtn">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>13</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>10</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/tg/tg/up.png);
</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>26</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>上一机房</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>26</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>下一机房</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QToolButton" name="downBtn">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>13</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>10</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/tg/tg/down.png);</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>5</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::DelayedPopup</enum>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="powWid" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="dataWid" native="true">
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#dataWid{background-color: rgb(0,17,55);}</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="vcWid" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="barWid" native="true">
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
